src/imagery/i.eodag/i.eodag.py

```diff
@@ -548,16 +550,14 @@ def filter_result(search_result, geometry, **kwargs):
     prefilter_count = len(search_result)
     area_relation = kwargs["area_relation"]
     minimum_overlap = kwargs["minimum_overlap"]
-    cloud_cover = kwargs["clouds"]
-    start_date = kwargs["start"]
-    end_date = kwargs["end"]
+    start_date = None
+    end_date = None
 
     # If neither a geometry is provided as a parameter
     # nor a vector map is provided through "options",
     # then none of the geometry filtering will take place.
```

Comment:
Why not? The region should be consider and the geometry filtering should happen with the region if no map is provided. Region can be as big or as small as user wants, so why not using it the same way map is used? Doesn't get_aoi fallback to get_bb if no vector is provided?

Reply:
> Doesn't get_aoi fallback to get_bb if no vector is provided?
Right, yes it does. I will fix that.